<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TATRC-17323" name="testCases.Admin.Simulation.SimulatorProgrammingPlatformAddEditDelete" tests="1" failures="1" timestamp="07 Nov 2019 13:29:04 GMT" time="45.351" errors="0">
  <testcase name="execute" time="45.351" classname="testCases.Admin.Simulation.SimulatorProgrammingPlatformAddEditDelete">
    <failure type="java.lang.AssertionError" message="Errors - Programming Platform Already Exists expected [Simulator Platform already exists.] but found []">
      <![CDATA[java.lang.AssertionError: Errors - Programming Platform Already Exists expected [Simulator Platform already exists.] but found []
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at testCases.Admin.Simulation.SimulatorProgrammingPlatformAddEditDelete.performErrorValidation(SimulatorProgrammingPlatformAddEditDelete.java:172)
	at testCases.Admin.Simulation.SimulatorProgrammingPlatformAddEditDelete.execute(SimulatorProgrammingPlatformAddEditDelete.java:93)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- execute -->
</testsuite> <!-- testCases.Admin.Simulation.SimulatorProgrammingPlatformAddEditDelete -->
